### DISCUSSION TITLE
lost my xch

### DISCUSSION THREAD

**Discussion by mahdibasir** [Chia-Network/chia-blockchain#16533] [Support]
I had 2 xch in my wallet.
After some time, I opened my wallet and my xch is missing.
please follow up

My wallet number is 1143080087
Thanks for this

**Comment by BrandtH22** [comment]
Hey @mahdibasir , the first thing I would try is a resync (Gui settings \ advanced resync wallet button then reopen the client and wait for sync). If this does not reveal the xch and/or if it shows transactions where the xch was sent then unfortunately it might be the case that your keyset has been compromised.

If the keyset is compromised I recommend reviewing the how to be a hard target article we have available here:
https://www.chia.net/2021/05/28/securing-your-chia-how-to-be-a-hard-target/
